pandas/core/arrays/boolean.py

```diff
@@ -492,6 +491,144 @@ def _values_for_argsort(self) -> np.ndarray:
         data[self._mask] = -1
         return data
 
+    def any(self, *, skipna: bool = True, axis: int | None = 0, **kwargs):
```

Comment:
where is this doc-string now?

Reply:
Looks like its gone now but was present here in 1.2.4.  We now share the method with some parent class, so wouldn't surprise me if this can be de-duplicated too